web/server/README.md

```diff
@@ -57,6 +57,59 @@ The API requests are serviced as follows:
 - `badges` gives access only to the badges API calls.
 - `management` gives access only to the management API calls.
 
+### Enabling TLS support
+
+
+Netdata since version 1.16 supports encrypted HTTP connections to the web server and encryption of the data stream between a slave and a master. 
+Inbound unix socket connections are unaffected, regardless of the SSL settings.  
+To enable SSL, provide the path to your certificate and private key in the `[web]` section of `netdata.conf`:
+
+```
+[web]
+	ssl key = /etc/netdata/ssl/key.pem
+	ssl certificate = /etc/netdata/ssl/cert.pem
+```
+
+Both files must be readable by the netdata user. For a master/slave connection, only the master needs these settings. 
```

Comment:
If any of those files is not readable/invalid the behavior is to fall back to http (even with `Force`). This should be documented imo.

Reply:
Good catch. here is the document with the update, please use this one @thiagoftsm 
[README.txt](https://github.com/netdata/netdata/files/3228172/README.txt)

Reply:
When any file is not readable, the library won't load, so we will always have http.

Reply:
I updated now!